viewer_clients/cmdline_client/cmd_line_client.py

```diff
@@ -41,63 +44,74 @@ def default(self, obj):
         d.update(obj.__dict__)
         return d
 
+
 def handle_auth_requests(args):
     session = session_manager.SessionManager_Client()
+
+    auth_token = session.getToken(args.host, args.port)
+
     auth_client = authentication_helper.ThriftAuthHelper(args.host,
                                                          args.port,
                                                          '/Authentication',
-                                                         session.getToken(
-                                                             args.host,
-                                                             args.port))
-
+                                                         auth_token)
     try:
         handshake = auth_client.getAuthParameters()
+
+        if not handshake.requiresAuthentication:
+            print("This server does not require privileged access.")
```

Comment:
Here and some other places: is print appropriate? When to use print and when to use LOG?